### DISCUSSION TITLE
Add columns in UI using Configure button does not work

### DISCUSSION THREAD

**Discussion by ziggekatten** [netbox-community/netbox#6866] [Q&A]
### NetBox version

v2.11.10

### Python version

3.8

### Steps to Reproduce

1. Click on Devices
2. Click on the "Configure" button
3. In the dialog box, mark any column you would like to add in the list in "Available columns"
4. Click "Add columns" 

### Expected Behavior

The columns should be added in the "Selected columns"

### Observed Behavior

Nothing happens when pressing "Add columns" and therefore the columns does not get added to the UI
Update: It's not possible to remove columns either.

This is not just related to devices, it is a generic issue in "Configure"

**Comment by jeremystretch** [comment]
I'm not able to replicate this either locally or on the [public demo instance]() running v2.11.10. It's likely an issue with your browser or installation. Try force-refreshing your browser and/or re-running `manage.py collectstatic`.

**Comment by ziggekatten** [comment]
It seems to be a problem with Chrome. It works with MS Edge and Firefox. Running Chrome: 92.0.4515.107

No errors in the development console. And it's not related to my browser only as my colleagues see the same thing. Weird.

**Comment by jeremystretch** [comment]
I'm also unable to reproduce on Chrome 92.0.4515.107. Please disable any installed extensions and try again.

Moving this to a discussion as it appears to be a local issue.

**Reply by candlerb** [reply]
Works for me too with Chrome (92.0.4515.107, under macOS 10.4.6)

I suggest you open the developer console (triple-dot, More Tools, Developer Tools) and try again.  Then look at "Console" and "Network" tabs.
